### PR TITLE
Recognize UltraVnc and TightVNC server by security type

### PR DESCRIPTION
**Pull request by badda71**
Closes https://github.com/LibVNC/libvncserver/issues/505

### CODE REVIEW

**Comment by badda71** [comment]
this PR fixes https://github.com/LibVNC/libvncserver/issues/505

File: libvncclient/rfbproto.c [hunk 1]
```diff
@@ -983,26 +993,6 @@ InitialiseRFBConnection(rfbClient* client)
   if ((major==rfbProtocolMajorVersion) && (minor>rfbProtocolMinorVersion))
     client->minor = rfbProtocolMinorVersion;
 
-  /* UltraVNC uses minor codes 4 and 6 for the server */
```
Comment (bk138) [inline libvncclient/rfbproto.c:986]: Those should be left as-is for compat with older servers.

File: libvncclient/rfbproto.c [hunk 2]
```diff
@@ -1206,7 +1196,6 @@ InitialiseRFBConnection(rfbClient* client)
   return TRUE;
 }
 
-
```
Comment (bk138) [inline libvncclient/rfbproto.c:1209]: Please don't add unrelated whitespace.

**Review by bk138 — CHANGES_REQUESTED** [review]

**Comment by bk138** [comment]
Looks good, thanks for the contribution!

**Pull request merged**